package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metamask/metamask-design-system",
-  "version": "2.0.0",
+  "version": "3.0.0",
```

Comment:
Create release branch updates the major version of the repo for every release

Reply:
Is this really the third release of this monorepo...?

Reply:
Yes, we haven't done many releases. We're getting questions about the migration and implementation of Tailwind in the extension, so we're working through some POCs and migration scripts based on that.

Reply:
Oh ok. Well that tells me that everything has been working well so far :)